plugins/module_utils/redfish_utils.py

```diff
@@ -2236,7 +2236,7 @@ def _insert_virt_media_payload(options, param_map, data, ai):
                 payload[param] = options.get(option)
         return payload
 
-    def virtual_media_insert_via_patch(self, options, param_map, uri, data):
+    def virtual_media_insert_via_patch(self, options, param_map, uri, data, image_only):
```

Comment:
For backwards compatibility, I think this should have default value `False`:
```suggestion
    def virtual_media_insert_via_patch(self, options, param_map, uri, data, image_only=False):
```